templates/webapi/main/index.html.ep

```diff
@@ -80,6 +80,25 @@
                 <%= help_popover('Help for <i>Full screen</i>' => '<p>Show builds in full screen mode</p>')
                 %>
             </div>
+
+            <div class="form-group" id="filter-interval-content">
+                <strong>
+                    Auto refresh
+                    <%= help_popover('Help for <i>Auto refresh</i>' => '<p>Allows to update the data of this page automatically</p>') %>
+                </strong>
+
+                <input value="0" name="interval" type="radio">
+                <label for="filter-interval">no refresh</label>
```

Comment:
But now the `for` attributes don't refer to a valid ID anymore.

Reply:
yes, new commit incomming